anthem/lyrics/modules.py

```diff
@@ -18,12 +18,21 @@ def uninstall(ctx, module_list):
         raise AnthemError(u'Cannot uninstall modules. See the logs')
 
 
-def update_translations(ctx, module_list):
+def update_translations(ctx, module_list, overwrite=False):
```

Comment:
This is a big non-backward compatible change, the `overwrite` default value was `True` before.
I think it makes more sense to have a `False` value by default so a dev has to explicitly enable `overwrite`.
But you should add a big, bold warning in the releases notes about it :)

Reply:
what about make it without default value, so all will be forced choose explicitly?

Reply:
good catch, I expected it to be False by default even before...
+1 for prominent release note + log its value w/ ctx.log_line